test/validators.test.js

```diff
@@ -13736,6 +13736,20 @@ describe('Validators', () => {
     });
     test({
       validator: 'isLicensePlate',
+
+      args: ['en-SG'],
+      valid: [
+        'SGX 1234 A',
+        'SGX-1234-A',
+        'SGB1234Z',
+      ],
+      invalid: [
+        'sg1234a',
+        'invalidlicenseplate',
+        '4578',
+        '',
+        'GJ054GH4785',
+      ],
       args: ['en-PK'],
```

Comment:
nit: can fix the duplicate `args` key in the tests?

Reply:
Fixed the duplicate args

Reply:
Hi @profnandaa can you review it?